tests/js/program-exit.bash

```diff
@@ -0,0 +1,35 @@
+#! /bin/bash
```

Comment:
where do these come from? Is the file header correct?

Reply:
Those were written by Wes a long time ago.

Reply:
I meant the file header docs ;-)